index.html

```diff
@@ -285,6 +379,70 @@
       </div>
     </div>
 
+    <!-- Invest -->
+    <div class="invest">
+      <div class="invest__container">
+        <div class="invest__title">We Invest in Your<br>Education</div>
+        
+        <div class="invest__text-area">
+          <div class="invest__text-area-title">What is ISA?</div>
+          <div class="invest__text-area-text">
+            Income Share Agreement (ISA) means that our success completely depends on your<br>
+            professional and financial growth. According to ISA students pay only when their<br>
+            salary exceed a certain monthly amount, and the fee is 20% of monthly gross salary<br>
+            until the entire tuition fee is not covered.
+          </div>
+        </div>
+
+        <div class="invest__text-area">
+          <div class="invest__text-area-title">What is the tuition fee?</div>
+          <div class="invest__text-area-text">
+            Tuition fee for the Coding Bootcamp depends on the Bootcamp, from 850.000 AMD<br>
+            for the QA Bootcamp till 1.800.000 AMD for the Machine Learning Bootcamp, but you<br>
+            start paying only when your salary reaches 200.000 AMD. If you want to pay in<br>
+            advance, you will get a huge discount. Payments must be made by the time the total<br>
+            amount is made. It means, faster your growth is, earlier you finish your payments.
+          </div>
+        </div>
+
+        <div class="invest__text-area invest__last-area">
+          <div class="invest__text-area-title">Does it mean, that I will get a job anyway?</div>
+          <div class="invest__text-area-text">
+            We guarantee your employment. It means, that if you're not employed in 4 months<br>
+            after the graduation, you have no financial obligations. From the very beginning we<br>
+            attract leading employers to these bootcamps to follow your success and consider<br>
+            you as a strong junior level developer candidate.
+            <br>
+            <br>
+            <br>
+            Find more information about the ISA and other organizational details in the F.A.Q.<br>
+            section.
+          </div>
+        </div>
+        
+      </div>
+    </div>
+
+    <!-- Consultation -->
+    <div id="cons" class="cons">
```

Comment:
`cons` is not a good variable name. Cons mean disadvantage, `consultation` is better one

Reply:
fixed